### PR TITLE
Add missing Qwen2-VL monkey patch test

### PR DESCRIPTION
**Pull request by tyler-romero**
## Summary
Add a monkeypatch test for Qwen2-VL.

Also fixes and re-enables Qwen2-VL multimodal tests

## Testing Done

- Hardware Type: <BLANK>
- [x] run `make test` to ensure correctness
- [x] run `make checkstyle` to ensure code style
- [x] run `make test-convergence` to ensure convergence

### CODE REVIEW

File: test/convergence/test_mini_models_multimodal.py [hunk 1]
```diff
@@ -140,6 +140,7 @@ def preprocess_function(examples):
             padding="max_length",
             truncation=True,
             max_length=1024,  # longer than for text-only b/c images require quite a few tokens
+            return_tensors="pt",
```
Comment (tyler-romero) [inline test/convergence/test_mini_models_multimodal.py:143]: unrelated fix

**Comment by tyler-romero** [comment]
All tests passing, except for the gemma2 tests that are failing on main

**Review by lancerts — APPROVED** [review]
